Source/Core/DolphinWX/Frame.cpp

```diff
@@ -1426,6 +1426,64 @@ void CFrame::ParseHotkeys()
 	{
 		State::Load(g_saveSlot);
 	}
+	if (IsHotkey(HK_TOGGLE_STEREO_SBS))
+	{
+		if (g_Config.iStereoMode != STEREO_SBS)
+		{
+			if (g_Config.sPostProcessingShader == "dubois")
```

Comment:
The reason I included the postprocessing shader (specifically anaglyph "dubois") in these other hotkeys was the fact that just enabling a stereoscopy mode via g_config.iStereoMode does not disable the anaglyph pp shader. This lead to side-by-side and top/bottom stereoscopy with false anaglyph colors.
It's an assurement that only one stereoscopy mode is selected when you click the hotkey. And it only disables the anaglyph mode, other pps's are available and unaffected.